maths/volume.dart

```diff
@@ -0,0 +1,100 @@
+import 'dart:math' show pi;
+
+import 'package:test/test.dart';
+
+/* 
+ ** Program to calculate the volume of the shapes
+ ** Reference: https://www.cuemath.com/measurement/volume
+*/
+
+/* Calculate the volume for a Cuboid */
+double volCuboid(double l, double b, double h) {
+  return l * b * h;
+}
+
+/* Calculate the volume for a Cube */
+double volCube(double a) {
+  return power(a, 3);
+}
+
+/* Calculate the volume for a Cone */
+double volCone(double r, double h) {
+  return pi * power(r, 2) * h / 3;
+}
+
+/* Calculate the volume for a Cylinder */
```

Comment:
lets not use single letter variables like `r` or `h`. can we use proper variable names like `radius` or `height`